Rakefile

```diff
@@ -27,7 +26,7 @@ namespace :test do
       Rake::Task["ci:setup:testunit"].execute
     end
     puts "Running Rails tests"
-    Rake::Task["test:run"].execute
+    Rake::Task["test"].execute
```

Comment:
I have changed `test:run` to `test` and added `rake test:all` in .travis.yml to include coveralls task and jasmine tests. Although, all tests stop running once an error or failure shows up but it doesn't shows up the warnings. Is it ok @jywarren?

Reply:
Didn't you say that's currently the way it works (unfortunately 😛 )? Can you open an issue to resolve that, but if it's not worse than the current state we can not block this PR because of it?

Thanks, @Souravirus !!

Reply:
ok let me check how it works with `rails test -d` once.